README.md

```diff
@@ -128,13 +128,15 @@ pkgman install cmake devel:libsdl2 devel:libsdl2_mixer devel:libsdl2_ttf devel:l
 ### Compiling on 32 bit Haiku
 ```
 cd build
-cmake -DCMAKE_C_COMPILER=gcc-x86 -DCMAKE_CXX_COMPILER=g++-x86 -DBINARY_RELEASE=ON ..
+setarch x86 #Switch to secondary compiler toolchain (GCC8+)
+cmake -DBINARY_RELEASE=ON ..
 cmake --build . -j $(nproc)
 ```
 ### Compiling on 64 bit Haiku
+No setarch required, as there is no secondary toolchain on x86_64, and the primary is GCC8+
 ```
 cd build
-cmake ..
+cmake -DBINARY_RELEASE=ON ..

```

Comment:
Do the debug builds not work on Haiku?

Reply:
AFAIK it works, but may i ask why is this information relevant in this case?

Reply:
The build instructions are mostly meant as a "how to start developing", so here having debug features enabled would be a benefit. If people then wan to build something like a release build they should see the CMake section where the different build types are described.

I believe that Haiku has prebuilt packages for DevilutionX already?

Reply:
But the 32 bit section already sports this cmake switch. If you wish i can remove it from both.

Reply:
Yes, please. It was a previous oversight, and the only one that had it. Also thank you for updating this.